### ISSUE TITLE
Using screenshot mode in a universal app

### ISSUE THREAD

**Issue by joshbaltzell** [Simbul/baker#434]
We are moving our iPad book to a univeral app and I assumed there would be some way to do screenshots for retina iPhone, standard iPhone, Retina iPad and standard iPad the way you can handle landscape and portrait now.

I looked a little closer and downloaded the latest code and there isn't anything that i can see. Surely someone else has made a universal app with screenshots right? Am I just missing the setting?

**Comment by folletto** [comment]
We should dig into this a little more deeper but at this point:
1. The rendering should work correctly without any setup on iPhone and iPad
2. The PRE-rendering isn't there yet, we didn't have any request on that feature and was behind schedule.

So, I think you are referring to the pre-rendering, right? :)

**Comment by joshbaltzell** [comment]
Yeah, pre-rendering. We will look closer, maybe we can just make the code edit and submit it. Otherwise we can make a separate app I guess.

**Comment by joshbaltzell** [comment]
Okay, we wrote some really simple code to handle it. After we know if is all working I'll contribute it.

My implementation required two separate JSON variables, which I think it a bad idea. I'd like a solution that can load screenshots for iPhone, iPad, iPhone Retina, and iPhone Retina. Maybe it should be handled by file names like landscape and portrait work?

**Comment by folletto** [comment]
I think so, but could you put here the `book.json` example so I can have a look? :)
